### ISSUE TITLE
JOSS publication

### ISSUE THREAD

**Issue by lsoucasse** [FormingWorlds/PROTEUS#177]
We aim at publishing the PROTEUS software in JOSS. This should happen once we incorporate Aragog and publish at least a first release of PROTEUS as a python package.

The paper consists in a short markdown file (example [here](https://joss.readthedocs.io/en/latest/example_paper.html)) containing a summary, a statement of interest and appropriate references. This file need to be hosted in the Github repository and compiled using this [GH action](https://github.com/marketplace/actions/open-journals-pdf-generator). It is also approciated to have a couple of tutorials, highlighting use cases and documenting how it works.

The review process is open and happens on Github, see [here](https://github.com/openjournals/joss-reviews). Submission requirements are listed [here](https://joss.readthedocs.io/en/latest/submitting.html).

**Comment by timlichtenberg** [comment]
Indeed, this is a clear plan to do in the first half of 2025. I suggest we discuss the approximate schedule and steps do before this submission in one of the next PROTEUS meetings, and then add these steps and timeline here as a guideline.

**Comment by timlichtenberg** [comment]
To achieve before we can:

- [ ] PyPi installation needs to work from scratch
- [ ] Extend the documentation to include running instructions for PyPi version and developer install
- [ ] Create a minimum example Juypter notebook for running the code

**Comment by timlichtenberg** [comment]
Currently anticipated timeline for JOSS publication:

~March: Write Draft of Paper
~April: Submit to JOSS

**Comment by timlichtenberg** [comment]
Update on JOSS backlog, will take a while.. https://github.com/openjournals/joss-reviews/issues/8758#issuecomment-3482952373

**Comment by timlichtenberg** [comment]
New update to the JOSS paper: https://github.com/openjournals/joss-reviews/issues/8758. Good news: the paper is moving along. Bad news: JOSS has significantly updated their editorial requirements due to increasing AI usage in software development. We will have to significantly rework the paper due to this. Right now it is the question whether this has to happen before or after we can get the first round of reviews...
